full-service/src/db/transaction_log.rs

```diff
@@ -2123,4 +2160,226 @@ mod tests {
             assert_eq!(tx_log.status(), TxStatus::Pending);
         }
     }
+
+    #[async_test_with_logger]
+    async fn test_lowest_pending_block_index(logger: Logger) {
+        let mut rng: StdRng = SeedableRng::from_seed([20u8; 32]);
+
+        let db_test_context = WalletDbTestContext::default();
+        let wallet_db = db_test_context.get_db_instance(logger.clone());
+        let known_recipients: Vec<PublicAddress> = Vec::new();
+        let mut ledger_db = get_test_ledger(5, &known_recipients, 12, &mut rng);
+
+        let first_account_key = random_account_with_seed_values(
+            &wallet_db,
+            &mut ledger_db,
+            &[70 * MOB, 75 * MOB],
+            &mut rng,
+            &logger,
+        );
+        let first_account_id = AccountID::from(&first_account_key);
+
+        let second_account_key = random_account_with_seed_values(
+            &wallet_db,
+            &mut ledger_db,
+            &[80 * MOB],
+            &mut rng,
+            &logger,
+        );
+        let second_account_id = AccountID::from(&second_account_key);
+
+        // Build a first transaction from first_account
+        let mut pooled_conn = wallet_db.get_pooled_conn().unwrap();
+        let conn = pooled_conn.deref_mut();
```

Comment:
consider this comment and the lines immediately following it.
The comment talks about building a transaction, but the next two lines deal with creating a database connection.
Then there is a blank line, which often communicates a change in thought  or logic

one will mentally group these 3 lines together as the same idea, _somewhat_ independent of the rest of the code
```suggestion
        let mut pooled_conn = wallet_db.get_pooled_conn().unwrap();
        let conn = pooled_conn.deref_mut();
```

Reply:
removed this code from the PR